src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1039,6 +1039,7 @@ private void OnAsyncTaskStateChanged(DynamoScheduler sender, TaskStateChangedEve
         public void Dispose()
         {
             EngineController.TraceReconcliationComplete -= EngineController_TraceReconcliationComplete;
+            EngineController.RequestCustomNodeRegistration -= EngineController_RequestCustomNodeRegistration;
```

Comment:
Thank you for make sure we unsubscribe. Things like this take much more time to identify later and do memory analysis